### PR TITLE
-update read me for activity log

### PR DESCRIPTION
**Pull request by ntthanh1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -284,6 +284,44 @@ src\Support\Traits\LastModifiedBy.php
...
+The `ActivityLoggable` trait provides functionality for logging activities and communication logs related to a model.
+ 
+Methods
+-------
+
+*   **`getActivityLogEmails(): array`**: Get the emails associated with activity logs.
+*   **`activities(): Collection`**: Get the collection of activities associated with the model.
+*   **`getModelChanges(?array $modelChangesJson = null): string`**: Get the model changes as a formatted string.
+*   **`getModelChangesJson(bool $allowCustomAttribute = false): array`**: Get the model changes as an array of JSON. If `$allowCustomAttribute` is date return of `getModelChanges` the activity will only track the of mention in `getModelChanges` 
+*   **`prepareModelChange($attribute, $from, $to): array`**: Prepare the model change data.
```
Comment (lionslair) [inline README.md:298]: I have no idea what the three parameters here are and what I am meant to pass to this. When do you use this? Before `createActivityLog` ?

File: README.md [hunk 2]
```diff
@@ -313,7 +349,52 @@ class Order extends Model
...
+namespace App\Observers;
+
+use App\Models\Order;
+use Dcodegroup\ActivityLog\Models\ActivityLog;
+
+class OrderObserver
+{
+    public function created(Order $model): void
+    {
+    $model->createActivityLog([
```
Comment (lionslair) [inline README.md:365]: In the code you have

```php
   $diff = data_get($description, 'diff', []);
```

But no example has this.

Are there only three parameters to pass in `Title` `Description` and `Diff`?

File: README.md [hunk 3]
```diff
@@ -313,7 +349,52 @@ class Order extends Model
...
+            'description' => '',
+        ]);
+    }
+}
+
+```
+In addition, we can add activity log wherever we want the model
+```php
+    $model->createActivityLog([
+            'type' => \Dcodegroup\ActivityLog\Models\ActivityLog::TYPE_COMMENT
```
Comment (lionslair) [inline README.md:393]: What are all the different types and when do you use each?

**Review by lionslair — CHANGES_REQUESTED** [review]
See comments

File: README.md (hunk 2 above)
Comment (ntthanh1) [reply README.md:365]: I don't think we need the 'diff' column anymore. We previously made it available to support search functionality in search terms. However it was never implemented @lionslair , @andrewsirianni

File: lang/en/activity-log.php [hunk 4]
```diff
@@ -28,6 +28,8 @@
     ],
     'actions' => [
         'create' => ' created a ',
+        'update' => ' updated a ',
+        'delete' => ' delete a',
```
Comment (andrewsirianni) [inline lang/en/activity-log.php:32]: This should be `deleted a`

File: lang/en/activity-log.php [hunk 5]
```diff
@@ -28,6 +28,8 @@
...
         'add_comment' => ' add a comment',
```
Comment (andrewsirianni) [inline lang/en/activity-log.php:33]: `added a comment`

**Review by andrewsirianni — CHANGES_REQUESTED** [review]

**Comment by lionslair** [comment]
Have opened a PR which may change some of the README updates 

https://github.com/DCODE-GROUP/laravel-activity-log/pull/40

File: README.md (hunk 1 above)
Comment (ntthanh1) [reply README.md:298]: Thank you, it was redundant, I just deleted it

**Review by lionslair — APPROVED** [review]

**Pull request merged**